### DISCUSSION TITLE
Table 9, Panel 1

### DISCUSSION THREAD

**Discussion by LSSchadler** [cssi-pi-community/2022-meeting#19] [Panel 1 roundtable (Monday morning)]
Test

**Comment by bkoz37** [comment]
The scientific software community needs a mechanism for sustained funding for long-term maintenance of scientific software developed e.g. in CSSI or similar programs under grants that last only a few years. The idea is to create a new funding sub-program at NSF that would award funding based on quantifiable continued usage metrics of software. For instance, if a software package that was developed in an Elements program is widely used by the community, this is represented by a large and increasing number of citations and Github stars. These metrics can be directly converted to an (annual / biannual) award amount from this new program to support maintenance and further development of software infrastructure. This will incentivize creation of software that is used by a wide community and achieves wide impact, and will create a predictable mechanism for maintaining such software.
